egs/wsj/s5/steps/data/augment_data_dir.py

```diff
@@ -117,16 +134,35 @@ def AugmentWav(utt, wav, dur, fg_snr_opts, bg_snr_opts, fg_noise_utts, \
             + start_times_str + " " + snrs_str + " - - |"
     return new_wav
 
-def CopyFileIfExists(utt_suffix, filename, input_dir, output_dir):
-    if os.path.isfile(input_dir + "/" + filename):
-        dict = ParseFileToDict(input_dir + "/" + filename,
+# This function generates a new id from the input id
```

Comment:
It's good that you are adding documentation, but let's do it with a doc-string, e.g.

```
def Foo(string):
     """ Foo is a function that does nothing.
               'string' is expected to be of type str, but is otherwise ignored.
          This function returns None.
     """
      return None
```